index.d.ts

```diff
@@ -810,6 +810,16 @@ declare namespace Moleculer {
 		options?: GenericObject
 	}
 
+	interface ReplCommand {
```

Comment:
Repl options have been migrated to the types files in `moleculer-repl` via moleculerjs/moleculer-repl#62.  If changes are necessary to the repl types, please create a PR in that repo instead of this one.